include/pingcap/kv/RegionClient.h

```diff
@@ -36,15 +36,15 @@ struct RegionClient
     {}
 
     // This method send a request to region, but is NOT Thread-Safe !!
-    template <typename T>
-    auto sendReqToRegion(Backoffer & bo,
-                         std::shared_ptr<T> req,
+    template <typename T, typename U, typename V>
```

Comment:
how about naming as typename REQ, RESP for better readibility?

Reply:
Done.